source/hierarchy.txt

```diff
@@ -12,22 +12,210 @@ Organization, Project, and Deployment Hierarchy
+   :header-rows: 1
+   :widths: 20 40
+
+   * - {+service+} Hierarchy Level
+     - Description
+
+   * - (Optional) Paying Organization
+     - One organization can be a paying organization for other
+       organization(s). The MongoDB account team must enable the paying organization when you establish the |service| subscription. A paying organization lets you set up :atlas:`cross-organization billing <billing/#std-label-cross-org-billing>` to share a
+       billing subscription across multiple organizations.
+
+   * - Organization
+     - Invoicing aggregates and generates at this level. Line items go
+       to cluster level and below. An organization often maps to a
+       business unit (BU) or application.
+
+   * - Project
+     - Security configuration for the data plane (including database
+       clusters and other data services) occurs at this level,
+       including network security. A project often maps to an application and environment (for example: Customer Portal app -
+       Production environment). For each project, there is one dedicated VPC created in GCP, and one VPC/VNet per region in AWS
+       and Azure. 
+
+   * - {+Cluster+}
+     - |service| provisions each {+cluster+} within the dedicated 
+       VPC/VNet(s) for a project. The security configuration is shared
+       among {+clusters+} in the project, except for user roles and authorization, which you can apply to actions on a {+cluster+},
+       database, and collection level.
+
+**DIAGRAM**
+
+Suggested Org and Project Hierarchies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _project-hierarchy-1:
+
+Suggested Hierarchy 1: Fewer Atlas Organizations
+````````````````````````````````````````````````
+
+This hierarchy, which creates fewer |service| organizations, might be useful if you have common teams and permissions across the business unit, and less than the (raiseable) limit of 250 projects per organization. 
+
+**DIAGRAM TO REPLACE TEXT**
+
+- Atlas Paying Org - Acme Inc
+
+  - Atlas Org: Consumer Products Business Unit
+
+    - Project: Consumer Customer Portal - Production (environment) 
+    - Project: Consumer Customer Portal - Test 
+    - Project: Consumer Customer Portal - Dev 
+    - Project: Product Mgmt App - Production 
+    - Project: Product Mgmt App - Test 
+    - Project: Product Mgmt App - Dev 
+
+  - Atlas Org: Corporate Products BU
+
+.. _project-hierarchy-2:
+
+Suggested Hierarchy 2: More Atlas Organizations
+```````````````````````````````````````````````
+
+This hierarchy creates a relatively large number of |service|
+organizations: one per application or scrum team. This hierarchy might
+be useful if each of your teams is fairly independent, they don't share people or permissions within a business unit, or they want to buy credits themselves through the cloud provider marketplace. There is
+no paying organization in this hierarchy.
+
+**DIAGRAM TO REPLACE TEXT**
+
+- Atlas Org: Consumer Customer Portal - paying at this level too
```

Comment:
Q: what does "paying at this level too" refer to?

Reply:
It means that each Org has its own billing (rather than all rolling up to one Paying org). But this text will be removed entirely when the diagram is ready and the diagram replaces it, so really just notes to understand without a diagram for now